lms/templates/student_account/enterprise_sidebar.html

```diff
@@ -0,0 +1,13 @@
+<div id="enterprise-content-container" class="enterprise-content">
+    <div class="centered-div">
+        % if enterprise_logo_url:
+            <img src="${enterprise_logo_url}" alt="" class="enterprise-logo">
+        % endif
+        <h2>
+            ${enterprise_branded_welcome_string}
+        </h2>
+        <h2>
+            ${platform_welcome_string}
+        </h2>
+    </div>
+</div>
```

Comment:
My thinking here is that even if decide to spin up an Enterprise IDA, we can also always install edx-enterprise as a python package in LMS to pull in the pieces of the Enterprise app that LMS needs to provide Enterprise customizations.

As a general rule, I think it makes sense to have the Enterprise-specific code live in the edx-enterprise repo. IDAs can pull this code in by installing the edx-enterprise package. This is mutually exclusive from the decision to a separate Enterprise service.